shared/transforms/xccdf-addremediations.xslt

```diff
@@ -109,7 +116,35 @@
     </xsl:call-template>
   </xsl:variable>
 
-  <xsl:value-of select="$rep4"/>
+  <xsl:variable name='newline'><xsl:text>
+</xsl:text></xsl:variable>
+
+  <xsl:variable name="rep5">
+    <xsl:variable name="platform_condition">
+      <xsl:choose>
+        <xsl:when test="$rule/xccdf:platform">
+          <xsl:choose>
+            <xsl:when test="$platform_cpe = 'machine'">
+              <xsl:value-of select="concat('when:  # Bare-metal-only rule, not applicable for containers', $newline, '    - ansible_virtualization_role == &quot;guest&quot;', $newline, '    - ansible_virtualization_type == &quot;docker&quot;')"/>
+            </xsl:when>
+            <xsl:otherwise>
+              <xsl:value-of select="concat('# Error ensuring that the platform is applicable - unknown platform CPE spec encountered: &quot;', $platform_cpe, '&quot;')"/>
+            </xsl:otherwise>
+	  </xsl:choose>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:value-of select="'# This rule is applicable for all platforms'"/>
```

Comment:
I think this is not needed to be emphasized, it is kind of implicit.

Reply:
OK